docs/keys.md

```diff
@@ -166,3 +166,18 @@ have my handlers attached:
 ```python
 SHREKLESS_ALT = KC.LALT.clone()
 ```
+
+You can also refer to a key by index:
+* KC['A']
+* KC['NO']
+* KC['LALT']
+
+or the get function which has an optional argument of `default`. `default` is
+be returned if the key is not found (`None` unless otherwise specified):
+* KC.get('A')
+* KC.get('NO', None)
+* KC.get('NOT DEFINED', KC.RALT)
```

Comment:
```suggestion
Or the `KC.get` function which has an optional default argument, which will
be returned if the key is not found (`default=None` unless otherwise specified):
* `KC.get('A')`
* `KC.get('NO', None)`
* `KC.get('NOT DEFINED', KC.RALT)`
```